api/types/src/transaction.rs

```diff
@@ -523,6 +555,38 @@ pub struct BlockMetadataTransaction {
     pub timestamp: U64,
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize, Object)]
+pub struct DummySystemTransaction {
+    nonce: u64,
+}
+
+/// A system transaction.
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize, Union)]
+pub enum SystemTransactionInner {
+    DummyTopic(DummySystemTransaction),
```

Comment:
why? rust can't have empty enums?

Reply:
rust can, but i'm not sure about all those derives. I can try.